src/hclib-deque.c

```diff
@@ -39,32 +39,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hclib-internal.h"
 #include "hclib-atomics.h"
 
-void deque_init(deque_t *deq, void *init_value) {
+#if 0 // UNUSED
```

Comment:
This 2-line change kept my diff smaller—but I agree we should delete this, as well as all the other unused/disabled code. There is *a lot* of it.